Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -198,39 +198,16 @@ First implementation.
 <ul>
-<li>
-When the plant enabling signal <code>on</code> changes from <code>false</code> to <code>true</code>, one chiller is enabled.
-</li>
-<li>
-When the total cooling load <code>QLoa</code> exceeds 80 percent (adjustable) of one chiller&apos;s nominal capacity 
-<code>QChi_nominal</code>, a second chiller is enabled.
-</li>
-<li>
-When the total cooling load <code>QLoa</code> drops below 60 percent (adjustable) of one chiller&apos;s nominal capacity 
-<code>QChi_nominal </code>(i.e., 30 percent each chiller), the second chiller is disabled. 
-</li>
-<li>
-When the plant enabling signal <code>on</code> changes from <code>true</code> to <code>false</code>, 
-the operating chiller is disabled.</li>
-<li>
-Parameter <code>tWai</code> assures a transitional time is kept between each operation.
-</li>
+<li>When the plant enabling signal <span style=\"font-family: Courier New;\">on</span> changes from <span style=\"font-family: Courier New;\">false</span> to <span style=\"font-family: Courier New;\">true</span>, one chiller is enabled. </li>
+<li>When the total cooling load <span style=\"font-family: Courier New;\">QLoa</span> exceeds 80 percent (adjustable) of one chiller&apos;s nominal capacity <span style=\"font-family: Courier New;\">QChi_nominal</span>, a second chiller is enabled. </li>
+<li>When the total cooling load <span style=\"font-family: Courier New;\">QLoa</span> drops below 60 percent (adjustable) of one chiller&apos;s nominal capacity <span style=\"font-family: Courier New;\">QChi_nominal </span>(i.e., 30 percent each chiller), the second chiller is disabled. </li>
+<li>When the plant enabling signal <span style=\"font-family: Courier New;\">on</span> changes from <span style=\"font-family: Courier New;\">true</span> to <span style=\"font-family: Courier New;\">false</span>, the operating chiller is disabled.</li>
+<li>Parameter <span style=\"font-family: Courier New;\">tWai</span> assures a transitional time is kept between each operation. </li>
 </ul>
-<p><br>
-It is assumed that both chillers have the same capacity of <code>QChi_nominal</code>.
-</p>
-<p>
-Note: This model can be used for plants with two chillers with or without waterside econimizer (WSE). 
-For plants with WSE, extra control logic on top of this model needs to be added.
-</p>
-<p>
-<img src=\"modelica://Buildings/Resources/Images/Experimental/DHC/CentralPlants/Cooling/Controls/ChillerStage.png\" alt=\"State graph\"/>. 
-</p>
+<p><br>It is assumed that both chillers have the same capacity of <span style=\"font-family: Courier New;\">QChi_nominal</span>. </p>
+<p>Note: This model can be used for plants with two chillers with or without waterside econimizer (WSE). For plants with WSE, extra control logic on top of this model needs to be added. </p>
+<p><img src=\"C:/Users/17206/Desktop/Stage control.tif\"/>. </p>
```

Comment:
Please use Modelica path for the image. Refer to my former model for an example.

Reply:
Please add alt text for the image as I did before.

Reply:
Please add alt text for the image like I did before.

Reply:
Done.